torch/csrc/cuda/Module.cpp

```diff
@@ -439,6 +439,31 @@ PyObject * THCPModule_memorySnapshot(PyObject *_unused, PyObject *noargs)
   END_HANDLE_TH_ERRORS
 }
 
+PyObject * THCPModule_cudaSetSyncDebugMode(PyObject * _unused, PyObject * arg){
+  HANDLE_TH_ERRORS
+  THPUtils_assert(THPUtils_checkLong(arg), "invalid argument to set_warn_on_synchronization");
```

Comment:
update name

Reply:
nit: TORCH_WARN_ONCE when this is called that the feature is a prototype and may not detect all CUDA syncs (yet)